src/shims/unix/linux/epoll.rs

```diff
@@ -51,7 +51,6 @@ pub struct EpollEvent {
     // The file descriptor struct associated with this epoll_event.
     pub weak_file_descriptor: WeakFileDescriptor,
     pub events: u32,
-    // TODO: pointer support, test that fails to have pointer.
```

Comment:
As Ralf mentioned, change it to a FIXME for using a Scalar with some info about the rust libc crate only allowing `u64`, but c libc docs saying it's a union that can also be a pointer

Reply:
ah you already did